extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/SparkRangerAdminPlugin.scala

```diff
@@ -135,7 +135,8 @@ object SparkRangerAdminPlugin extends RangerBasePlugin("spark", "sparkSql")
     val pos = if (hasLen) ", 5" else ""
     val upper = s"regexp_replace($expr, '[A-Z]', 'X'$pos)"
     val lower = s"regexp_replace($upper, '[a-z]', 'x'$pos)"
-    val digits = s"regexp_replace($lower, '[0-9]', 'n'$pos)"
+    val chinese = s"regexp_replace($lower, '[\u4e00-\u9fa5]', 'x'$pos)"
```

Comment:
What's the behavior on other popular databases/OLAP systems?
And I remember that UTF-8 reserved range for CJK, can we extend it to cover CJK?

Reply:
> What's the behavior on other popular databases/OLAP systems? And I remember that UTF-8 reserved range for CJK, can we extend it to cover CJK?
Spark needs to access the metastore to query data, and the metadata information in the metastore is stored in mysql.In mysql, the character set is also utf-8
<img width="594" alt="image" src="https://user-images.githubusercontent.com/33471639/225561693-42487689-dad6-4560-a563-ffcb8a0985ec.png">

CJK can be covered by the following regular expression
<img width="1435" alt="image" src="https://user-images.githubusercontent.com/33471639/225562034-4ee77e2d-1dd1-4777-b182-2add3e39aa85.png">